### PR TITLE
Default includes

### PR DESCRIPTION
**Pull request by HurricanKai**
Enables OSX Support & Resolves default Includes

### CODE REVIEW

File: src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs [hunk 1]
```diff
@@ -57,6 +59,51 @@ public IEnumerable<Symbol> ScrapeXML(XmlDocument document)
...
+            if (VisualStudioResolver.TryGetVisualStudioInfo(out var info))
+            {
+                if (info.UcrtIncludes.Length > 0)
+                {
+                    foreach (var include in info.UcrtIncludes)
+                    {
+                        yield return include;
+                    }
+                }
+                else
```
Comment (Perksey) [inline src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs:96]: Don't we want both?

Comment (HurricanKai) [reply src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs:96]: I think this is the ucrt vs msvc code? GH lost track of the snippet.
Not an expert, just quickly googled online, and there I got the impression ucrt is just a newer rewrite of C(++) standard headers, while msvc is the older stuff?

Comment (Perksey) [reply src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs:96]: I think the naming is misleading, the ucrt array is likely to only contain windows sdk headers and the MSVC tools contains the standard C headers.

Comment (HurricanKai) [reply src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs:96]: Ah, I see. Will change in that case :+1:

File: src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs [hunk 2]
```diff
@@ -57,6 +63,51 @@ public IEnumerable<Symbol> ScrapeXML(XmlDocument document)
         return visitor.Visit(bindings);
     }
 
+    private string GetXCodeSdkPath()
+    {
+        var process = new Process();
+        process.StartInfo = new ProcessStartInfo("xcrun", $"--show-sdk-path --sdk {_options.Value.XcodeSdk}")
```
Comment (Perksey) [inline src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs:69]: This might be null

**Pull request merged**